tests/queries/0_stateless/02836_file_diagnostics_while_reading_header.sh

```diff
@@ -6,6 +6,6 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 
 FILENAME="${CLICKHOUSE_TMP}/test.csv"
 
-printf 'Bad\rHeader\n123\n' > "${FILENAME}"
+printf 'Bad,Header\n123\n' > "${FILENAME}"
```

Comment:
In the original issue, it is mentioned that the CRLF character (`\r\n`) is used as the end-of-line character. Should we also add a test for it?

Reply:
We already support CRLF at and of file for CSV format for a long time and already have tests for it